src/elm/Community.elm

```diff
@@ -232,29 +232,24 @@ settingsQuery symbol =
     Query.community { symbol = symbolToString symbol } settingsSelectionSet
 
 
-logoUrl : String -> Maybe String -> String
-logoUrl ipfsUrl maybeHash =
-    case maybeHash of
+logoUrl : Maybe String -> String
+logoUrl maybeUrl =
+    case maybeUrl of
         Nothing ->
-            logoPlaceholder ipfsUrl
+            ""
 
-        Just hash ->
-            if String.isEmpty (String.trim hash) then
-                logoPlaceholder ipfsUrl
+        Just url ->
+            if String.isEmpty (String.trim url) then
+                ""
 
             else
-                ipfsUrl ++ "/" ++ hash
+                url
 
 
-logoBackground : String -> Maybe String -> Html.Attribute msg
-logoBackground ipfsUrl maybeHash =
+logoBackground : Maybe String -> Html.Attribute msg
+logoBackground maybeHash =
     Html.Attributes.style "background-image"
-        ("url(" ++ logoUrl ipfsUrl maybeHash ++ ")")
-
-
-logoPlaceholder : String -> String
-logoPlaceholder ipfsUrl =
-    ipfsUrl ++ "/QmXuf6y8TMGRN96HZEy86c8N9aDseaeyuCQ5qVLqPyd8Ld"
+        ("url(" ++ logoUrl maybeHash ++ ")")
```

Comment:
Any ideas on how we should handle this placeholder? Maybe we could have the API return the information, and we'd fetch it once and cache. If we're going to hardcode a URL it might be better to just serve it as a static asset in the frontend itself (maybe as an SVG?)

Reply:
+1 to adding it to the project image folder

Reply:
Done. I can't find a place to test it, but I followed the way other icons are shown in the application so it should work just fine :)